README.md

```diff
@@ -8,15 +8,15 @@ This four week course is designed to introduce attendees to central concepts in
 
 ## Schedule
 
-- Week 1: Introduction and Conceptual Overview; Machine Learning and Experimental Design
-- Week 2: Supervised Learning via Regression
-- Week 3: Supervised Learning via Classification
-- Week 4: Unsupervised Learning via Dimensionality Reduction, Clustering, and Transfer Learning
+- Class 1: Introduction and Conceptual Overview of Machine Learning Concepts
```

Comment:
I like adding links to the markdown file in each line, so that clicking on Class 1 takes you to the file.